optimum/intel/neural_compressor/modeling_base.py

```diff
@@ -113,6 +123,71 @@ def _from_pretrained(
         trust_remote_code: bool = False,
         **kwargs,
     ):
+        quantization_config = kwargs.pop("quantization_config", None)
+
+        if hasattr(config, "quantization_config"):
+            if config.quantization_config is None:
+                logger.warning(
+                    "Quantization_config loading failed. If you want to load saved "
+                    "low bit model, please check your quantizate_config.json."
+                )
+            else:
+                logger.info("quantization_config: {}".format(config.quantization_config))
+
+                try:
+                    from neural_compressor.transformers.models.modeling_auto import (
+                        _BaseINCAutoModelClass,
+                    )
+
+                    warnings.warn(
+                        "Weight only quantization model loading provided by intel_extension_for_transformers is deprecated and it is provided by INC now.",
+                        DeprecationWarning,
+                    )
```

Comment:
could this be determined from the model itself (that the model was quantized through ITREX) ?

Reply:
added a check here https://github.com/huggingface/optimum-intel/pull/880/commits/8bddf205e74e0e749ad46d7f19abc7fa1a5db64e : checking the quantization configuration (if present + matching algorithm parameter)